Northstar.CustomServers/mod/scripts/vscripts/sh_progression.nut

```diff
@@ -169,8 +169,13 @@ void function UpdateCachedLoadouts_Threaded()
 	// below here is just making all the menu models update properly and such
 
 	#if UI
-	uiGlobal.pilotSpawnLoadoutIndex = GetPersistentSpawnLoadoutIndex( GetUIPlayer(), "pilot" )
-	uiGlobal.titanSpawnLoadoutIndex = GetPersistentSpawnLoadoutIndex( GetUIPlayer(), "titan" )
+	entity UIPlayer = GetUIPlayer()
+	
+	if (!IsValid ( UIPlayer ) )
```

Comment:
Minor nitpicking with formatting, the space is missing, so it should be `if ( !IsValid ( UIPlayer ) )`

Reply:
Really it should be `if ( !IsValid( UIPlayer ) )`